scripts/setup-linter-dependencies.py

```diff
@@ -0,0 +1,156 @@
+import json
```

Comment:
would you add a short file comment here demonstrating how to invoke the script?

Reply:
I made a proposition in the last commit.

Reply:
very good